src/components/Service.jsx

```diff
@@ -46,25 +69,42 @@ const ScreenSplit = ({ data }) => {
       <Grid item xs={12}>
         {data.coords.latitude ? mapDisplay() : <Divider />}
       </Grid>
-      <Grid container style={{marginTop: "18px"}}>
+      <Grid container style={{ marginTop: "18px" }}>
         <Grid item xs={12} md={6} style={styles.gridItem}>
           <Typography variant="h6" gutterBottom>
             Description:
           </Typography>
-          <Typography data-cy="description" variant="body2" component="p" className={classes.text}>
+          <Typography
+            data-cy="description"
+            variant="body2"
+            component="p"
+            className={classes.text}
+          >
             {data.description}
           </Typography>
         </Grid>
         <Grid item xs={12} md={6} style={styles.gridItem}>
           <Typography variant="h6" gutterBottom>
             Contacts:
           </Typography>
-          <Typography data-cy="contacts" variant="body2" component="p" style={styles.contacts} className={classes.text}>
+          <Typography
+            data-cy="contacts"
+            variant="body2"
+            component="p"
+            style={styles.contacts}
+            className={classes.text}
+          >
             {data.telephone && `phone: ${data.telephone}\n`}
             {data.email && `email: ${data.email}\n`}
             {data.address && `address: ${data.address}\n`}
             {data.postcode && `postcode: ${data.postcode}\n`}
-            {data.website && `website: ${data.website}\n`}
+            Website:
+            {data.website ? (
```

Comment:
change ? for && and then you dont need : undefined

Reply:
Fixed :)